apps/chat-e2e/src/tests/promptUsage.test.ts

```diff
@@ -83,7 +86,7 @@ dialTest(
       `Type "/the" in send message input and verify one prompt is shown`,
       async () => {
         await sendMessage.messageInput.fillInInput(searchTerm);
-        await sendMessagePromptListAssertion.assertPromptListOptions(
+        await sendMessagePromptListAssertion.assertPromptListIncludesOptions(
           promptNames.filter((p) =>
             p.toLowerCase().includes(searchTerm.substring(1)),
```

Comment:
Looks like an incorrect check. Why are we checking all the prompt names? We should check only the third one. If the UI behaviour is reversed and it shows only the "prompt 1" and "prompt 2", we will have the assertion passed

Reply:
done